docs/src/references/child-devices.md

```diff
@@ -16,22 +20,76 @@ Child-Device support allows to connect devices (e.g. sensors, actors, PLCs, any
 
 ## Principles
 
-Child-device support basically focuses on two aspects:
-   1) enable external-devices to access thin-edge APIs via network
-   2) equip thin-edge APIs to associate consumed/provided data with a child-device twin in the cloud
+Child-device support basically focuses on three aspects:
+   1) provision the child-device, i.e. making thin-edge and the cloud aware of the child-device (external or logical) 
+   2) thin-egde APIs provided on network, to enable external APIs to access them
+   3) associate consumed/provided data of a child-device (external or logical) to it's child-device twin in the cloud
 
-Each thin-edge API that supports child-devices applies to both aspects. As of now APIs as below have child-device support:
+### (1) Child-Device Provisioning
+
+A newly attached child-device (external or logical) must be once provisioned. The provisioning phase of a child-device includes:
+  1) creating cloud's child-device twin
+  2) declaring all supported capabilities of the child-device to its child device twin
+     (i.E. declaring all _supported operations_ and all _types_ per supported operation, as _config types_, _log types_, ...)
```

Comment:
This line is redundant with the next line. I'd just keep the next one.

Reply:
Removed from that PR, but to be considered in PR "requirements and use-cases" (#1195).
See more details in comment https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r955257236